.gitattributes

```diff
@@ -1,2 +1,5 @@
 data/kaggle-ground-truth/*.jpg filter=lfs diff=lfs merge=lfs -text
 data/kaggle-ground-truth/*.csv filter=lfs diff=lfs merge=lfs -text
+data/storysquad-ground-truth/*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
Since we have multiple datasets at this point a more generic pattern is probably appropriate. The pattern `data/*/*.png` for example would match every png in a nested folder under data.

Reply:
Working on this now